packages/devextreme/js/__internal/ui/context_menu/menu_base.ts

```diff
@@ -779,25 +778,24 @@ class MenuBase<
     return result;
   }
 
-  // @ts-expect-error ts-error
-  _updateSelectedItems(
```

Comment:
this._updateSelectedItems() is implemented in parent components. The selection module calls it on SelectionChanged when args.addedItemKeys.length or args.removedItemKeys.length is non-zero. I see that the options for this are not public in Menus, but if you are sure that all processing of selected items happens on the MenuBase side and the selection module shouldn’t perform any updates, we need to override _updateSelectedItems() with an empty function